### DISCUSSION TITLE
Search chapter in Bookmark

### DISCUSSION THREAD

**Discussion by 9Htang** [sumatrapdfreader/sumatrapdf#3740] [Ideas]
Maybe having a search bar at the top of the bookmarks to search for specific characters is useful                           :-)

**Comment by GitHubRulesOK** [comment]
It is a MS Windows form control so does respond like File Explorer to search just one key on visible entries.

Simple example 

Here I can press `E` or `A` as only those 2 options are in focus
![image](https://github.com/sumatrapdfreader/sumatrapdf/assets/12754913/c3cb5abc-d3ff-4110-b4c7-b941de6b0bdb)
However once I expand all the number of active hot keys now includes `G` `P` `L` & `F` with `A` and `G` now allowing for "cycling" around the multiple entries
![image](https://github.com/sumatrapdfreader/sumatrapdf/assets/12754913/7322200b-81dd-4f5a-a78e-48bd466dd72c)
